src/sass/components/govcy-text-inputs.scss

```diff
@@ -0,0 +1,132 @@
+  $border-radius : 1px;
+  $govcy-outline-color:yellow;
+  
+  
+    textarea:focus,
+    input[type="text"]:focus,
+    input[type="password"].focus,
+    input[type="datetime"]:focus,
+    input[type="datetime-local"]:focus,
+    input[type="date"]:focus,
+    input[type="month"]:focus,
+    input[type="time"]:focus,
+    input[type="week"]:focus,
+    input[type="number"]:focus,
+    input[type="email"]:focus,
+    input[type="url"]:focus,
+    input[type="search"]:focus,
+    input[type="tel"]:focus,
+    input[type="color"]:focus,
+    .uneditable-input:focus {   
+       border-color: gray; 
+      box-shadow: 0 5px 5px rgba(0, 0, 0, 0.075) inset, 0 0 8px rgba(126, 239, 104, 0.6);
+      outline: 5px solid $govcy-outline-color;
+      border-radius: $border-radius;
+    }
+
+  @mixin govcy-text-inputs-borders-fonts {
+
+    border-width: 3px;
+    border-color: gray;
+    border-radius: $border-radius;
+    font-size: x-large;
+  }  
+
+.govcy-text-input-base {
+   
+    @extend .form-control;
+    @extend .form-text;
+    @include govcy-text-inputs-borders-fonts();
```

Comment:
I don't _think_ you need the brackets `()`
https://sass-lang.com/documentation/at-rules/mixin
Could be wrong though. My sass is a bit out of date